EnglishTranslation-master/ETC.tsv

```diff
@@ -11594,7 +11594,7 @@ ETC_20150511_011595	DoTimeAction:Picking Niole roots:1:WEEDING:None
 ETC_20150511_011596	Activate the 1st sealed device
 ETC_20150511_011597	Activate the 2nd sealed device
 ETC_20150511_011598	Activate the 3rd sealed device
-ETC_20150511_011599	FAQ : Durability & Potential{nl}1) Durability : Starts to decrease slowly when you play with equipped equipment.{nl}    When an item's durability is below a certain value, the item's icon will be displayed on the right side of the quick slot.{nl}   When the durability completely diminishes, you won't be benefit from the effects of your equipment.{nl}2) Potential : Decreases when you trade or enhance an equipment. The item with its Potential all depleted can't be traded or enhanced.
+ETC_20150511_011599	FAQ : Durability & Potential{nl}1) Durability : Starts to decrease slowly when you play with equipped equipment.{nl}    When an items durability is below a certain value, the items icon will be displayed on the right side of the quick slot.{nl}   When the durability completely diminishes, you won't be benefit from the effects of your equipment.{nl}2) Potential : Decreases when you trade or enhance an equipment. The item with its Potential all depleted can't be traded or enhanced.
```

Comment:
The previous is correct. It's the durability of an item, not the durability of an items.